Vic2ToHoI4/Source/V2World/Provinces/ProvinceFactory.cpp

```diff
@@ -27,6 +28,13 @@ Vic2::Province::Factory::Factory(std::unique_ptr<PopFactory>&& _popFactory): pop
 		 [this](const std::string& popType, std::istream& theStream) {
 			 province->pops.push_back(popFactory->getPop(popType, theStream));
 		 });
+	registerKeyword("flags", [this](std::istream& theStream) {
+		const auto assignments = commonItems::assignments{theStream}.getAssignments();
```

Comment:
did you not decide to ditch curlys?

Reply:
Perils of copy/pasting. Thanks for noticing!